coffee_github/src/repository.rs

```diff
@@ -318,6 +331,7 @@ impl From<&Github> for StorageRepository {
             url: value.url.to_owned(),
             plugins: value.plugins.to_owned(),
             branch: value.branch.to_owned(),
+            commit: value.commit.to_owned(),
```

Comment:
yeah nice but it is the wrong place, you can always access to git to extract this information, the commit should be inside the plugin when we move out the plugin in another directory